src/test/docker/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Use a temporary layer for the build stage.
-FROM vitess/base:v17.0.2 AS base
+FROM vitess/base:v19.0.3 AS base
 
-FROM vitess/lite:v17.0.2
+FROM vitess/lite:v19.0.3
```

Comment:
There was a nasty bug in 19.0.3, so I would generally recommend 19.0.4+ (even though it may not be terribly relevant here).

Reply:
Thanks, updated!